components/NavBar.vue

```diff
@@ -58,6 +81,10 @@ export default {
     return {
       visible: 'visible',
```

Comment:
Can we set the initial state of the navbar to `hidden` so that it only appears on scrolling, not when you first load the page?

Reply:
> The mobile navbar looks great! However, for some reason about halfway between desktop size and mobile I cannot see a navbar or a hamburger menu, even when scrolling:

@rmehri01 Thanks for noticing that! Seems like it's happening when the screen size falls between (768, 1024). I confirmed that `visible` and `opacity` were set correctly when that happens, and I noticed that some sass file imported from bulma sets `navbar-menu` to `display: none` by default and with the min screen width of 1024px, they set it to flex which then will be visible. Should we look into this further in this PR or be a separate issue?

![Screen Shot 2020-08-19 at 8 36 00 AM](https://user-images.githubusercontent.com/28884850/90658320-4b519c00-e1f8-11ea-8408-ce70b939876c.png)
![Screen Shot 2020-08-19 at 8 38 40 AM](https://user-images.githubusercontent.com/28884850/90658333-4ee52300-e1f8-11ea-840f-a0d9c161ae1f.png)

Reply:
@alice-0-kim Hi, thanks for looking into that. To be honest, since it is on the other nw sites as well, we could just make an issue and look into it later it isn't a huge deal.

Reply:
> Can we set the initial state of the navbar to `hidden` so that it only appears on scrolling, not when you first load the page?

Just wanted to clarify, currently the nav bar fades out when the user scrolls down, did you want it to appear instead then?

Reply:
No, that part is fine. I'm saying just when the user first loads the page. So this is when they first load it now:

![image](https://user-images.githubusercontent.com/52933714/90661129-0b90b180-e204-11ea-97a1-a110871cdde0.png)

And this is how it would be if it were disabled initially:

![image](https://user-images.githubusercontent.com/52933714/90661170-19463700-e204-11ea-860e-0308274e203a.png)

This is because for the hero page we already have the nav sign so there isn't really a point in having both. The scroll up for navbar is still good for nav on the rest of the site!

Reply:
Oh okay that makes sense -- just updated the PR accordingly!

Reply:
Would it still be okay if we don't have the MLH badge hidden on load as well? It's part of the navbar

Reply:
> Would it still be okay if we don't have the MLH badge hidden on load as well? It's part of the navbar

For now yes because we don't know if we are sponsored by MLH or not LOL. We'll have to change it later anyways so don't worry about it.

Reply:
> Oh okay that makes sense -- just updated the PR accordingly!

Is there a way to make it only happen on desktop because I just noticed that mobile doesn't have the nav sign, sorry!

Reply:
> > Oh okay that makes sense -- just updated the PR accordingly!
> 
> Is there a way to make it only happen on desktop because I just noticed that mobile doesn't have the nav sign, sorry!

Done! Thanks for catching!

Reply:
@alice-0-kim Umm it's only showing the MLH banner on mobile not the hamburger menu?

![image](https://user-images.githubusercontent.com/52933714/90675941-692ef900-e218-11ea-80e8-c4adf67fbb28.png)

Reply:
Are you able to scroll horizontally? I noticed that the sponsor's images are too large and do not fit into the screen, so for smaller screen the hamburger menu tends to be dragged out of the screen, but it should be there. If that is not the case, do let me know!

Reply:
Oh yeah I see it now thanks!